routers/private/internal.go

```diff
@@ -0,0 +1,43 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package private
```

Comment:
Just realized there already is `modules/ssh`. Does it make any sense to combine the two?